src/algebra/monoid_algebra.lean

```diff
@@ -321,15 +331,16 @@ def single_one_alg_hom {A : Type*} [comm_semiring k] [semiring A] [algebra k A]
 { commutes' := λ r, by { ext, simp, refl, }, ..single_one_ring_hom}
 
 @[simp] lemma coe_algebra_map {A : Type*} [comm_semiring k] [semiring A] [algebra k A] [monoid G] :
-  (algebra_map k (monoid_algebra A G) : k → monoid_algebra A G) = single 1 ∘ (algebra_map k A) :=
+  ⇑(algebra_map k (monoid_algebra A G)) = single 1 ∘ (algebra_map k A) :=
 rfl
 
 lemma single_eq_algebra_map_mul_of [comm_semiring k] [monoid G] (a : G) (b : k) :
-  single a b = (algebra_map k (monoid_algebra k G) : k → monoid_algebra k G) b * of k G a :=
+  single a b = algebra_map k (monoid_algebra k G) b * of k G a :=
```

Comment:
I assume I should take this as a statement that coercious to a function via `coe` instead of `coe_fn` is a bad idea?

Reply:
The original code did coercion via `coe_fn` too but it was more explicit.